src/tests/comment.test.js

```diff
@@ -204,4 +205,79 @@ describe('comment tests', () => {
         done();
       });
   });
+
+  it('should display an error message in case a accomodation does not have a comment', (done) => {
```

Comment:
@shemaeric1234 Just because there're no comments/feedback to an accommodation, doesn't mean that an error happened, it's just the comments list is empty. I suggest using HTTP 204 No Content.